libs/core/langchain_core/tracers/log_stream.py

```diff
@@ -210,12 +209,11 @@ def __init__(
         self.exclude_types = exclude_types
         self.exclude_tags = exclude_tags
 
-        send_stream: Any
-        receive_stream: Any
```

Comment:
I think this syntax with square brackets doesn't exist in older py

Reply:
hmm i think it's there for classes? I think the issue originally was that the source library was using it with a function rather than a class.

Evidence is that all unit tests pass